be/src/formats/parquet/group_reader.cpp

```diff
@@ -214,7 +214,8 @@ vectorized::ChunkPtr GroupReader::_create_read_chunk(const std::vector<int>& col
 void GroupReader::collect_io_ranges(std::vector<SharedBufferedInputStream::IORange>* ranges, int64_t* end_offset) {
     int64_t end = 0;
     for (const auto& column : _param.read_cols) {
-        auto& rg = _row_group_metadata->columns[column.col_idx_in_parquet].meta_data;
+        auto schema_node = _param.file_metadata->schema().get_stored_column_by_idx(column.col_idx_in_parquet);
```

Comment:
Better to add some comments to explain why get metadata from row_group_meta rather than file_meta.

Reply:
> Better to add some comments to explain why get metadata from row_group_meta rather than file_meta.

Because we need some row group meta (such as data_page_offset and total_compressed_size) to make up the shared buffer range of this group. If use file meta, we still need to locate row group by the group index, it is unnecessary.

Reply:
yeah, better to write this to the code and make others know

Reply:
> yeah, better to write this to the code and make others know

OK